families/burrow_evm/src/sawtooth_burrow_evm/cli/main.go

```diff
@@ -122,7 +124,13 @@ func decodeFileOrArg(arg, enc string) (b []byte, err error) {
 	}
 
 	if enc == "hex" {
-		return sdk.MustDecode(arg), nil
+		bytes, err := hex.DecodeString(arg)
+		if err != nil {
+			return nil, &processor.InternalError{ Msg: fmt.Sprintf(
```

Comment:
The errors defined in `sawtooth_sdk/processor` are only for use by a transaction processor.  Since this is error is just being printed for the user to see, it is fine to just use `fmt.Errorf()` which creates a generic error with the string you give it.